src/components/stub_comps/srof/cime_config/config_component.xml

```diff
@@ -4,6 +4,10 @@
 
 <entry_id>
 
+  <description mustmatch="SROF">
```

Comment:
Please make this consistent with the v3 format.

Reply:
fixed.